libnavui-dropin/src/main/res/layout/mapbox_info_panel_header_active_guidance_layout.xml

```diff
@@ -14,17 +14,18 @@
         android:layout_height="wrap_content"
         android:layout_marginTop="20dp"
         app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintEnd_toStartOf="@id/endNavigation"
+        app:layout_constraintEnd_toStartOf="@id/endNavigationContainer"
         app:layout_constraintStart_toStartOf="parent" />
 
-    <com.mapbox.navigation.ui.base.view.MapboxExtendableButton
-        tools:style="@style/DropInStyleExitButton"
-        android:id="@+id/endNavigation"
-        android:layout_width="72dp"
-        android:layout_height="52dp"
-        android:layout_marginTop="26dp"
+    <FrameLayout
+        android:id="@+id/endNavigationContainer"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
         android:layout_marginEnd="10dp"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:layout_height="52dp"
+        tools:layout_width="72dp" />
```

Comment:
We should set a max width to FrameLayout because `tripProgress` is constrained to to this `endNavigationContainer`. If at runtime the end user assigns layout params to this container such that the end navigation takes up the entire space, in that case tripProgress will go out of the screen. Hence to avoid that lets assign `maxWidth=72dp`

Same applies to all other `FrameLayouts` added in this PR 👇🏻